.gitignore

```diff
@@ -19,3 +19,4 @@ tmp*
 attrs.docset
 attrs.tgz
 Justfile
+t.py
```

Comment:
What's this for?

Reply:
The place I tested your types. ;) Hatch is a bit weird about what files it packages, so I've stopped using global gitignores / .git/info/exclude for projects that I release myself.